docs/how-to-build-with-ddn/with-snowflake.mdx

```diff
@@ -0,0 +1,302 @@
+---
+sidebar_position: 6
+sidebar_label: With Snowflake
+description: "Learn the basics of Hasura DDN and how to get started with a Snowflake instance."
+keywords:
+  - hasura ddn
+  - graphql api
+  - getting started
+  - guide
+  - snowflake
+---
+
+import Prereqs from "@site/docs/_prereqs.mdx";
+
+# Get Started with Hasura DDN and Snowflake
+
+## Overview
+
+This tutorial takes about twenty minutes to complete. You'll learn how to:
+
+- Set up a new Hasura DDN project
+- Connect it to a Snowflake instance
+- Generate Hasura metadata
+- Create a build
+- Run your first query
+
+Additionally, we'll familiarize you with the steps and workflows necessary to iterate on your API.
+
+This tutorial assumes you're starting from scratch, but you can easily follow the steps if you already have data seeded;
+Hasura will never modify your source schema.
+
+<Prereqs />
+
+## Tutorial
+
+### Step 1. Authenticate your CLI
+
+```sh title="Before you can create a new Hasura DDN project, you need to authenticate your CLI:"
+ddn auth login
+```
+
+This will launch a browser window prompting you to log in or sign up for Hasura DDN. After you log in, the CLI will
+acknowledge your login, giving you access to Hasura Cloud resources.
+
+### Step 2. Scaffold out a new local project
+
+```sh title="Next, create a new local project:"
+ddn supergraph init my-project && cd my-project
+```
+
+Once you move into this directory, you'll see your project scaffolded out for you. You can view the structure by either
+running `ls` in your terminal, or by opening the directory in your preferred editor.
+
+### Step 3. Initialize your Snowflake connector
+
+```sh title="In your project directory, run:"
+ddn connector init my_snowflake -i
+```
+
+```plaintext title="The CLI will ask you for a connection string in the JDBC URL format, e.g.:"
+jdbc:snowflake://<account-identifier.<region>.snowflakecomputing.com?user=YOUR_USERNAME&&password=YOUR_PASSWORD&db=YOUR_DATABASE&warehouse=YOUR_WAREHOUSE&schema=YOUR_SCHEMA&role=YOUR_ROLE
+```
```

Comment:
A string I have working does not have the region. I think it is optional. Also the schema, role, warehouse I think are optional too. I know you're getting confirmation from Gavin. 
eg:
`"jdbc:snowflake://ABCDEF-ABC4925.snowflakecomputing.com/?user=SEPARO&db=SNOWFLAKE_SAMPLE_DATA&password=my_password"`

Reply:
There's also a double ampersand before password in the existing string `&&password=YOUR_PASSWORD`